Linker/Linker.Android/Linker.Android.csproj

```diff
@@ -156,5 +156,16 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </LinkDescription>
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="DotfuscatorConfig.xml" />
+  </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
+  <PropertyGroup>
+    <DotfuscatorXamarinConfigFileName>DotfuscatorConfig.xml</DotfuscatorXamarinConfigFileName>
+    <DotfuscatorXamarinCliPath>C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\Extensions\PreEmptiveSolutions\DotfuscatorCE\dotfuscatorCLI.exe</DotfuscatorXamarinCliPath>
```

Comment:
so assumption here is the person getting this project must have the enterprise version of VS 2019 installed at this path?

Reply:
That's a good question and I wasn't sure how to solve this. At the moment, yes, the expectation is that if you don't have the dotfuscator installed it won't work

Reply:
ok guess will have to see if there's a way to do it differently (note: i know the guide references the absolute path like you have)

Reply:
Three lines up it uses $(MSBuildExtensionsPath) in the Import element. Can we see if that works when specifying the DotfuscatorXamarinCliPath?